packages/@react-aria/live-announcer/src/LiveAnnouncer.tsx

```diff
@@ -20,17 +20,28 @@ let node = null;
 let clearTimeoutId = null;
 const LIVEREGION_TIMEOUT_DELAY = 1000;
 
+type TAriaLive = 'assertive' | 'off' | 'polite' | undefined;
```

Comment:
we don't use the 'T' naming convention, AriaLive is fine, see https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/interactions/src/useFocusVisible.ts#L21
The aria-live definition in TS is `'aria-live'?: 'off' | 'assertive' | 'polite';` I don't think we need undefined there and we can use the optional when it is truly optional

Reply:
Ahh i see okay i'll change these to normal convention. 

🤔 hmm yeah right no need to have undefined in the union. will fix it.